tests/lang/s01_basics/s06_data_types/t5_enumeration.nim

```diff
@@ -1,13 +1,13 @@
 discard """
 description: '''
-An enumeration defines a type with a closed set of values. These values are
+An enumeration defines a type with a closed set of values. These values
 have an identifier, are ordered, and have corresponding integer and string
 values.
 '''
 target: "c cpp js"
 """
 
-block:
+block enumDefaultIntValues:
```

Comment:
All other named blocks used `snake_case` for naming - Ideally I would probably use `` `Stroped but human-readable name` `` if stropping didn't have random issues with dashes